<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.2.2 (Python 3.9.0 on win32)" generated="20210207 11:30:21.347" rpa="true">
<suite id="s1" name="Tasks" source="F:\BISMILLAH_ROBOT\ROBOCORP\maria-weekly-task-rpa-\Tasks\tasks.robot">
<test id="s1-t1" name="Dowload excel file of sales report and fill the web page">
<kw name="Open Robotsparebin intranet">
<kw name="Open Available Browser" library="RPA.Browser.Selenium">
<doc>Opens the first available browser in the system in preferred order, or the
given browser (``browser_selection``).</doc>
<arguments>
<arg>${robotsparebin_web_url}</arg>
</arguments>
<msg timestamp="20210207 11:30:22.367" level="INFO">Creating an instance of the Chrome WebDriver.</msg>
<msg timestamp="20210207 11:30:22.682" level="INFO">Creating an instance of the Chrome WebDriver.</msg>
<msg timestamp="20210207 11:30:22.689" level="INFO">Creating an instance of the Firefox WebDriver.</msg>
<msg timestamp="20210207 11:30:22.730" level="INFO">Creating an instance of the Firefox WebDriver.</msg>
<msg timestamp="20210207 11:30:22.738" level="INFO">Creating an instance of the Edge WebDriver.</msg>
<msg timestamp="20210207 11:30:22.776" level="INFO">Creating an instance of the Edge WebDriver.</msg>
<msg timestamp="20210207 11:30:22.784" level="INFO">Creating an instance of the Ie WebDriver.</msg>
<msg timestamp="20210207 11:30:22.800" level="INFO">Creating an instance of the Ie WebDriver.</msg>
<msg timestamp="20210207 11:30:22.807" level="INFO">Creating an instance of the Opera WebDriver.</msg>
<msg timestamp="20210207 11:30:22.820" level="INFO">Creating an instance of the Opera WebDriver.</msg>
<msg timestamp="20210207 11:30:22.827" level="INFO" html="yes">&lt;p&gt;Attempted combinations:&lt;/p&gt;&lt;div class="doc"&gt;&lt;table&gt;&lt;tr&gt;&lt;th&gt;Browser&lt;/th&gt;&lt;th&gt;Headless&lt;/th&gt;&lt;th&gt;Download&lt;/th&gt;&lt;th&gt;Error&lt;/th&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Chrome&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Chrome&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;Failed to download wedriver: HTTPSConnectionPool(host='www.googleapis.com', port=443): Max retries exceeded with url: /storage/v1/b/chromedriver/o/LATEST_RELEASE (Caused by NewConnectionError('&lt;urllib3.connection.HTTPSConnection object at 0x000001F91641DE50&gt;: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Chrome&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Chrome&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;Failed to download wedriver: HTTPSConnectionPool(host='www.googleapis.com', port=443): Max retries exceeded with url: /storage/v1/b/chromedriver/o/LATEST_RELEASE (Caused by NewConnectionError('&lt;urllib3.connection.HTTPSConnection object at 0x000001F9164ABFD0&gt;: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Firefox&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'geckodriver' executable needs to be in PATH. 
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Firefox&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;Failed to download wedriver: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/mozilla/geckodriver/releases/latest (Caused by NewConnectionError('&lt;urllib3.connection.HTTPSConnection object at 0x000001F91652BF10&gt;: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Firefox&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'geckodriver' executable needs to be in PATH. 
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Firefox&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;Failed to download wedriver: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/mozilla/geckodriver/releases/latest (Caused by NewConnectionError('&lt;urllib3.connection.HTTPSConnection object at 0x000001F91653BDC0&gt;: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Edge&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'MicrosoftWebDriver.exe' executable needs to be in PATH. Please download from http://go.microsoft.com/fwlink/?LinkId=619687
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Edge&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;Failed to download wedriver: HTTPSConnectionPool(host='developer.microsoft.com', port=443): Max retries exceeded with url: /en-us/microsoft-edge/tools/webdriver/ (Caused by NewConnectionError('&lt;urllib3.connection.HTTPSConnection object at 0x000001F916544AC0&gt;: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Edge&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'MicrosoftWebDriver.exe' executable needs to be in PATH. Please download from http://go.microsoft.com/fwlink/?LinkId=619687
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Edge&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;Failed to download wedriver: HTTPSConnectionPool(host='developer.microsoft.com', port=443): Max retries exceeded with url: /en-us/microsoft-edge/tools/webdriver/ (Caused by NewConnectionError('&lt;urllib3.connection.HTTPSConnection object at 0x000001F91654D880&gt;: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Ie&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'IEDriverServer.exe' executable needs to be in PATH. Please download from http://selenium-release.storage.googleapis.com/index.html and read up at https://github.com/SeleniumHQ/selenium/wiki/InternetExplorerDriver
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Ie&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;Failed to download wedriver: HTTPSConnectionPool(host='selenium-release.storage.googleapis.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('&lt;urllib3.connection.HTTPSConnection object at 0x000001F9165596A0&gt;: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Ie&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'IEDriverServer.exe' executable needs to be in PATH. Please download from http://selenium-release.storage.googleapis.com/index.html and read up at https://github.com/SeleniumHQ/selenium/wiki/InternetExplorerDriver
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Ie&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;Failed to download wedriver: HTTPSConnectionPool(host='selenium-release.storage.googleapis.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('&lt;urllib3.connection.HTTPSConnection object at 0x000001F9165644C0&gt;: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Opera&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'operadriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Opera&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;Failed to download wedriver: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/operasoftware/operachromiumdriver/releases/latest (Caused by NewConnectionError('&lt;urllib3.connection.HTTPSConnection object at 0x000001F91656E280&gt;: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Opera&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'operadriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Opera&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;Failed to download wedriver: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/operasoftware/operachromiumdriver/releases/latest (Caused by NewConnectionError('&lt;urllib3.connection.HTTPSConnection object at 0x000001F916576040&gt;: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/div&gt;</msg>
<kw name="Capture Page Screenshot" library="RPA.Browser.Selenium">
<doc>Takes a screenshot of the current page and embeds it into a log file.</doc>
<msg timestamp="20210207 11:30:22.839" level="INFO">Cannot capture screenshot because no browser is open.</msg>
<status status="PASS" starttime="20210207 11:30:22.838" endtime="20210207 11:30:22.839"></status>
</kw>
<msg timestamp="20210207 11:30:22.885" level="FAIL">BrowserNotFoundError: No valid browser found from: Chrome, Firefox, Edge, Ie, Opera</msg>
<status status="FAIL" starttime="20210207 11:30:22.358" endtime="20210207 11:30:22.885"></status>
</kw>
<status status="FAIL" starttime="20210207 11:30:22.358" endtime="20210207 11:30:22.885"></status>
</kw>
<kw name="Logout to Robotsparebin web page and close browser" type="teardown">
<doc>We logout user after all task done</doc>
<kw name="Click Element When Visible" library="RPA.Browser.Selenium">
<doc>Click element identified by ``locator``, once it becomes visible.</doc>
<arguments>
<arg>id:logout</arg>
</arguments>
<kw name="Capture Page Screenshot" library="RPA.Browser.Selenium">
<doc>Takes a screenshot of the current page and embeds it into a log file.</doc>
<msg timestamp="20210207 11:30:22.887" level="INFO">Cannot capture screenshot because no browser is open.</msg>
<status status="PASS" starttime="20210207 11:30:22.887" endtime="20210207 11:30:22.887"></status>
</kw>
<msg timestamp="20210207 11:30:22.933" level="FAIL">No browser is open.</msg>
<status status="FAIL" starttime="20210207 11:30:22.887" endtime="20210207 11:30:22.933"></status>
</kw>
<kw name="Close Browser" library="RPA.Browser.Selenium">
<doc>Closes the current browser.</doc>
<status status="PASS" starttime="20210207 11:30:22.933" endtime="20210207 11:30:22.933"></status>
</kw>
<status status="FAIL" starttime="20210207 11:30:22.886" endtime="20210207 11:30:22.933">No browser is open.</status>
</kw>
<tags>
<tag>accomplish maria task</tag>
</tags>
<status status="FAIL" starttime="20210207 11:30:22.356" endtime="20210207 11:30:22.934" critical="yes">BrowserNotFoundError: No valid browser found from: Chrome, Firefox, Edge, Ie, Opera

Also teardown failed:
No browser is open.</status>
</test>
<doc>Acomplish maria weekly task that consist to download a file and fill a web page</doc>
<status status="FAIL" starttime="20210207 11:30:21.349" endtime="20210207 11:30:22.937"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tasks</stat>
<stat pass="0" fail="1">All Tasks</stat>
</total>
<tag>
<stat pass="0" fail="1">accomplish maria task</stat>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="Tasks">Tasks</stat>
</suite>
</statistics>
<errors>
</errors>
</robot>
